<?xml version="1.0" encoding="UTF-8"?><testsuite errors="0" failures="4" name="com.gorest.datadriventests.UserDataDrivenTest" skipped="0" tests="9" time="6.55" timestamp="2023-01-11T19:52:19.824Z[Europe/London]"><testcase classname="com.gorest.datadriventests.UserDataDrivenTest" name="Data Driven test for creating multiple users"><failure message="1 expectation failed.&#10;Expected status code &lt;201&gt; but was &lt;422&gt;.&#10;" type="java.lang.AssertionError">1 expectation failed.
Expected status code &lt;201&gt; but was &lt;422&gt;.
</failure><system-err>1 expectation failed.
Expected status code &lt;201&gt; but was &lt;422&gt;.
&#13;
io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:636)&#13;
io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:122)&#13;
io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:130)&#13;
io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:117)&#13;
com.gorest.datadriventests.UserDataDrivenTest.createMultipleUsers(UserDataDrivenTest.java:27)&#13;
</system-err></testcase></testsuite>